### ISSUE TITLE
Add a public paper to his paper list

### ISSUE THREAD

**Issue by tdurieux** [gnieh/bluelatex#234]
We can now create public paper and thus contributors are not inevitably an author's or a reviewer.
I think it's a good idea to be able to save the public paper to his paper list because the ID of the paper is not easy to memorize.

**Comment by satabin** [comment]
I am not sure about this one… You may potentially have thousands of public papers on an instance, and the way of sharing such papers is (just like it used to be in the very first version of \BlueLaTeX) to send the URL to edit it by email or IM, or whatever.
@monperrus Any position on this?

**Comment by tdurieux** [comment]
I agree but my idea was to had a button in public paper which adds the paper to the user paper list.

**Comment by satabin** [comment]
OK I understand.
This will require some more work to be done on the server side.
